### PR TITLE
Use utils.command  to replace exec.command

### PR DESCRIPTION
**Pull request by cheyang**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/utils/exec.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+package utils
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// allowPathlist of safe commands
+var allowPathlist = map[string]bool{
```
Comment (TrafalgarZZZ) [inline pkg/utils/exec.go:26]: How about renaming it to `allowCommandList`?

File: pkg/utils/exec.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+			return
+		}
+		cmd = exec.Command(name, arg...)
+	}
+
+	return
+}
+
+// CheckCommandArgs is check string is valid in args
+func checkCommandArgs(arg ...string) (err error) {
```
Comment (TrafalgarZZZ) [inline pkg/utils/exec.go:51]: I think we can simplify this by using `strings.ContainsAny()`

File: pkg/utils/helm/helm.go [hunk 3]
```diff
@@ -93,7 +93,10 @@ func GenerateHelmTemplate(name string, namespace string, valueFileName string, c
 	// return syscall.Exec(cmd, args, env)
 	// 5. execute the command
 	log.V(1).Info("Generating template", "args", args)
-	cmd := exec.Command("bash", "-c", strings.Join(args, " "))
+	cmd, err := utils.SafeCommand("bash", "-c", strings.Join(args, " "))
```
Comment (TrafalgarZZZ) [inline pkg/utils/helm/helm.go:96]: This seems in a unused function? How about removing the function?

File: pkg/utils/exec.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// allowPathlist of safe commands
+var allowPathlist = map[string]bool{
+	// "helm":    true,
```
Comment (TrafalgarZZZ) [inline pkg/utils/exec.go:27]: Why not adding `ddc-helm` to the white-list?

File: pkg/utils/exec.go [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+	"kubectl": true,
```
Comment (TrafalgarZZZ) [inline pkg/utils/exec.go:28]: I think we can use `func init()` to find the kubectl and ddc-helm command path, and put them into the white list dynamically

File: pkg/utils/exec.go (hunk 4 above)
Comment (cheyang) [reply pkg/utils/exec.go:27]: Thanks, I forgot it.

File: pkg/utils/exec.go (hunk 2 above)
Comment (cheyang) [reply pkg/utils/exec.go:51]: I think it can indicate which char is illegalChar.

File: pkg/utils/helm/helm.go (hunk 3 above)
Comment (cheyang) [reply pkg/utils/helm/helm.go]: done

**Review by TrafalgarZZZ — APPROVED** [review]
/lgtm

**Pull request merged**